README-spec.md

```diff
@@ -88,6 +123,7 @@ User:
     * first_task_of_this_session_performed_at: timestamp.
     (default: start of time)
     * is_locked: (default: False)
+    * hourly_rate: float (default: None)
 
 Batch:
```

Comment:
Is_deleted should be added, so users wouldn't have to be completely deleted from db.

Reply:
I'm on the fence about this. It's a lot more maintenance to keep track of deleted stuff which is affecting the quality of everything, and leads to more possible bugs (not just from webdevs but also from scientists using the software.) Thus I want permanent delete because it's safer.

The only real benefit is keeping track of total money spent, because bad annotators cost money anyway. If you have a very simple proposal for how to add that, let's hear it, but otherwise let's skip it. There are other ways to track budget.